### PR TITLE
Update deps to reflect eslint config change and add lint script

### PR DESCRIPTION
**Pull request by mdholloway**
Updates the dependencies as required to accommodate a recent change
to the eslint config for node services.[1]  Fixes all lint errors.

Adds a lint script to package.json.

Fixes T181564.

[1] https://github.com/wikimedia/eslint-config-node-services/commit/c48908abb7fc6768bd890e63da685fb460c9cb6f

### CODE REVIEW

File: test/utils/logStream.js [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
 
-    function halt() {
-      if (end === null) {
-        end = log.length;
-      }
-    }
+        return {
+            halt,
+            get
+        };
```
Comment (d00rman) [inline test/utils/logStream.js]: Node complains about a syntax error here.

**Review by d00rman — CHANGES_REQUESTED** [review]
Thank you for looking into this @mdholloway! One comment in-lined, otherwise LGTM.

**Comment by mdholloway** [comment]
Thanks, @d00rman!  Should be fixed now.

**Comment by d00rman** [comment]
Oh right, it was because of the shorthand. Thank you @mdholloway !

**Review by d00rman — APPROVED** [review]

**Pull request merged**